### PR TITLE
Adding MySql howto-documentation and example DAG

### LINKED ISSUE #11918: Add how-to Guide for MySQL operators
**Description**

A guide that describes how to use all the operators for MySQL (https://github.com/apache/airflow/tree/master/airflow/providers/mysql/operators) would be useful.

Other guides are available:
https://airflow.readthedocs.io/en/latest/howto/operator/index.html

Source code for those guides are at:
https://github.com/apache/airflow/tree/master/docs/howto/operator

Are you wondering how to start contributing to this project? Start by reading our [contributor guide](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst)

Best regards,
Kaxil

### PR DESCRIPTION
**Pull request by jfmolano**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by jfmolano** [comment]
Dear @kaxil I rebased my repo as you told me but the checking process is still stuck at CI Build / Wait for PROD images (approx 2 hours). What should I do now?

**Comment by potiuk** [comment]
I think simply the queue was long due to many rebased PRs. But it should be ok now.

**Comment by potiuk** [comment]
Dont worry about the failing PROD build. It's being fixe d in master, but should have no effect for the review.

**Comment by jfmolano** [comment]
Oh, ok, that you @potiuk !! So all I have to do now is waiting until the PR is reviewed? I'm newby at open source contribution.

File: docs/howto/operator/mysql.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    under the License.
+
+
+
+.. _howto/operator:MySqlOperator:
+
+MySqlOperator
+=================
+
+Use the :class:`~airflow.operators.mysql_operator.MySqlOperator` to execute
```
Comment (kaxil) [inline docs/howto/operator/mysql.rst]:
```suggestion
Use the :class:`~airflow.providers.mysql.operators.MySqlOperator` to execute
```

File: docs/howto/operator/mysql.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+     - MySql port
+
+An example usage of the MySqlOperator is as follows:
+
+.. exampleinclude:: /../airflow/providers/mysql/example_dags/example_mysql.py
+    :language: python
+    :start-after: [START howto_operator_mysql]
+    :end-before: [END howto_operator_mysql]
+
+You can also use an external file to execute the Sql commands. Script folder must be at the same level as DAG .py file.
```
Comment (kaxil) [inline docs/howto/operator/mysql.rst]:
```suggestion
You can also use an external file to execute the SQL commands. Script folder must be at the same level as DAG.py file.
```

**Review by kaxil — COMMENTED** [review]
Looks good @jfmolano , I have added minor suggestions

**Comment by kaxil** [comment]
Once you accept those suggestions, this PR can be merged. Well done

**Comment by jfmolano** [comment]
Ok @kaxil suggestions accepted 😃 Thank you so much! I am really excited for making my first open source contribution! 🎉🎉🎉

**Review by kaxil — APPROVED** [review]
LGTM (Looks Good To Me). Let's wait for the CI Results 🤞

**Comment by jfmolano** [comment]
@kaxil Cool!! CI checks were OK. Now the merge is done automatically or should I do something else?

**Comment by kaxil** [comment]
Well done, congratulations @jfmolano

**Comment by jfmolano** [comment]
Thank you for all your help @kaxil I'm really happy! 🎉

**Pull request merged**